README.md

```diff
@@ -62,6 +62,73 @@ This starts a preview server on your machine, and watches all files for changes.
 
 The preview server supports hot reloading. Once the server is running on your machine, any changes you make to Markdown content will appear automatically in your browser within a few seconds. Note that changes to page frontmatter or site configuration require you to stop and start the preview server.
 
+## Windows 
+
+If you are on a Windows machine you may experience the following issues.
+
+### EOL Error
+Because Windows handles line endings differently than *nix operating systems, you may see the following error after running `yarn dev`:
+
+```
```

Comment:
You can add bash/shell markdown here.

Reply:
I considered that; I wasn't sure if that was appropriate since the text is just output and not bash commands.

Reply:
I think it's OK since this is the output you see in your console after running `yarn dev`.